core-api/src/controllers/permisson.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response } from "express";
+import { PermissonService } from "../services/permisson.service";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import Messages from "../utils/messages";
+
+export class PermissonController {
+    public static async createPermisson(req: Request, res: Response) {
+        const { name } = req.body;
+        if (!name) {
+            res.status(HttpStatus.BAD_REQUEST).json(new ServerError(
+                Messages.validation.data_needs_to_be_provided,
```

Comment:
This message is not suited. Please create another.

Reply:
CAN CLOSE